specs/grafana-integration.adoc

```diff
@@ -0,0 +1,140 @@
+in the tendrl app should be updated to support grafana integration.
+* All the related jobs in tendrl-copr, jenkins should also updated.
+
+
+=== Alternatives
+
+None
+
+
+=== Data model impact:
+
+None
+
+
+=== Impacted Modules:
+
+
+==== Tendrl/monitoring-integration impact:
+This is the new package added into tendrl app to dynamically configure
+grafana by using the json template with the given values or the retrieved
+values from etcd. The json will be used by the grafana service which will
+fetche the values from graphite to visualize the data. There will be a
+service which runs in the tendrl-server to perform this job.
+This package needs to be builded for tendrl.
+This package (rpm) should install all the required packages and the
+service file during installation.
+Reference Doc: https://github.com/Tendrl/specifications/pull/218
+
+
+==== Tendrl/node_agent impact:
+Node-monitoring will be integrated with node-agent. The services of
+both the components will be merged together. The changes should be
+updated into the spec file, after the merge of the pr for the
+spec: https://github.com/Tendrl/specifications/pull/219.
+Build scripts should be updated to support make-snapshot which should
+automatically update the release version as per fedora guidelines to
+integrate with centos-ci
+
+
+==== Tendrl/api impact:
+This package will be used for calling api to importing gluster and
+user identity management. The build script should be updated to support
+make-snapshot and automatic release version.
+
+
+==== Tendrl/gluster-integration impact:
+This package is now newly using belivet to fetch device details.
+This should be updated into the spec file along with other changes.
+Build scripts should be updated to support make-snapshot which should
+automatically update the release version as per fedora guidelines to
+integrate with centos-ci.
+
+
+==== Tendrl/dashboard impact:
+This package will be used to show the tendrl dashboard which will show the
+appropriate grafana dashboard based on user’s selection.
+This tendrl component is now renamed to tendrl-ui. This change should be
+updated everywhere in copr, jenkins and in the tendrl-ui spec files.
+Add required changes for make-snapshot support and automatic version
+change as per guidelines.
+
+
+==== Notifications/Monitoring impact:
+Remove all the build releated script for performance monitoring,
+node-monitoring from copr, jenkins ci.
+
+
+==== Tendrl/common impact:
+Update the scripts for automatic version change as per guidelines.
+
+
+==== Sds integration impact:
+None
+
+=== Security impact:
+None
+
+
+=== Other end user impact:
+None
+
+
+=== Performance impact:
+None
+
+
+=== Other deployer impact:
+None
+
+
+=== Developer impact:
+None
+
+
+=== Assignee(s):
+Timothy Asir Jeyasing
+
+
+=== Work Items:
+None
+
+
+== Dependencies:
+None
+
+== Testing:
+Test whether the package installation installs all the packages properly
+
+
+== Documentation impact:
+None
```

Comment:
Since we don't have documentation builds polished and restarted, a separate wikipage listing the installation steps of just grafana and monitoring-integration components would be good enough.